features/show_dishes.feature

```diff
@@ -5,17 +5,21 @@ Feature: user can see dishes and their info
 
   Background:
     Given the following category exists
-      | name        | description         |
-      | Japanese    | Japanese food       |
+      | name        | description |
+      | Thai        | Thai food   |
+    And the following menu exist
+      | name      |
+      | Everyday  |
     And the following dish category exists
       | name        |
       | Main        |
-    And the following dishes exists
-      | name        | description         | price               | pic_url                      | dish_category |
-      | Sushi rolls | Tasty Japanese food | 10                  | https://goo.gl/images/hpTGCV | Main          |
     And the following restaurants exist
-      | name        | address             | restaurant_category |
-      | Yappi       | Stockholm           | Japanese            |
+      | name        | address             | restaurant_category   |  description          | menu     |
+      | Yappi     | Stockholm           | Thai                  |  Some hip thai place  | Everyday |
```

Comment:
I see what you are doing here but I would create a separate step with a table setting menus for restaurants
```
And the following menus exist
  | name        | restaurant             |
  | Everyday  | Yappi                    |
```
And the step definition could be:
```
Given("the following menus exist") do |table|
    table.hashes.each do |hash|
      restaurant = Restaurant.find_by(name: hash[:restaurant])
      hash.delete(:restaurant)
      menu = FactoryGirl.create(:menu, hash)
    end
  end
```
It's basically the same thing but simpler and more readable, imo. This is not a blocker though.